src/boot/efi/boot.c

```diff
@@ -778,16 +778,16 @@ static bool menu_run(
                         size_t len = strnlen16(status, x_max - 1);
                         size_t x = (x_max - len) / 2;
                         status[len] = '\0';
-                        print_at(0, y_status, COLOR_NORMAL, clearline + x_max - x);
+                        print_at(0, y_status + 1, COLOR_NORMAL, clearline + x_max - x);
```

Comment:
Please add an explanation in the commit message what is being fixed. E.g., maybe an example or some other explanation.

I would like to test this patch, but since I'm not sure what the bug is, and how it's supposed to be fixed, that is hard.

Reply:
Hey,
thank you very much for your comment :)
The bug I encountered was that the status line was not correctly shown. It was off by one line. But it might be that I introduced this issue myself?